google-cloud-storage/lib/google/cloud/storage/service.rb

```diff
@@ -182,9 +181,8 @@ def list_default_acls bucket_name, user_project: nil
         ##
         # Creates a new default ACL.
         def insert_default_acl bucket_name, entity, role, user_project: nil
-          new_acl = Google::Apis::StorageV1::ObjectAccessControl.new(
-            { entity: entity, role: role }.delete_if { |_k, v| v.nil? }
-          )
+          param  = { entity: entity, role: role }.delete_if { |_k, v| v.nil? }
```

Comment:
lib/google/cloud/storage/service.rb:184:16: C: Layout/ExtraSpacing: Unnecessary spacing detected.
          param  = { entity: entity, role: role }.delete_if { |_k, v| v.nil? }
               ^
```suggestion
          param = { entity: entity, role: role }.delete_if { |_k, v| v.nil? }
```